pom.xml

```diff
@@ -85,6 +85,20 @@ THE SOFTWARE.
             <groupId>io.jenkins.plugins</groupId>
             <artifactId>ionicons-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>prism-api</artifactId>
+            <version>1.29.0-7-rc295.1064eb_c55f37</version>
+        </dependency>
+        <!-- For some reason need to explicitly depend on commons-text-api and commons-lang3-api for plugin-util-api to load -->
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>commons-text-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>commons-lang3-api</artifactId>
+        </dependency>
```

Comment:
When I use prism 1.29.0-8 as local dependency I do not see such a problem. Where exactly did you see the problem? I've seen that in the BOM builds Mark also had this problem.

```suggestion       
```

Reply:
unsure but lots of people were complaining about it for checks-api, and I added them there too:
https://github.com/jenkinsci/checks-api-plugin/issues/233

Although at least in checks api it actually used those dependencies.

Reply:
This fails injected tests from maven hpi plugin, if you try locally you will get:

```
[ERROR] Failures:
[ERROR]   InjectedTest.testPluginActive While testing design-library, plugin-util-api failed to start
```

```
Caused by: java.io.IOException: Failed to load: Plugin Utilities API Plugin (plugin-util-api 3.3.0)
 - Plugin is missing: commons-text-api (1.10.0-36.vc008c8fcda_7b_)
 - Plugin is missing: commons-lang3-api (3.12.0-36.vd97de6465d5b_)
	at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:988)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:555)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:177)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:305)
	at jenkins.model.Jenkins$5.runTask(Jenkins.java:1170)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:221)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:120)
	at jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:68)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
```